### ISSUE TITLE
General Hadean & Cerberi Issues/Suggestions

### ISSUE THREAD

**Issue by Malifious** [BlackDragonN001/FERemastered#100]
Attached are .txt files with all issues found between both. I went through and tested every unit and building under a variety of conditions to try and bait out any bugs that may be present.

I'm going to do a further break-down for both EDF and Scion next and then dive into campaign testing, which it looks like it already being attacked pretty heavily by other testers.

Any suggestions, comments, questions, concerns, or requests are appreciated and welcomed! 

[Cerberi Errors (General).txt](https://github.com/BlackDragonN001/FERemastered/files/4658792/Cerberi.Errors.General.txt)

[Hadean Errors (General).txt](https://github.com/BlackDragonN001/FERemastered/files/4658793/Hadean.Errors.General.txt)

**Comment by Malifious** [comment]
NOTE : Besides the Xerrakis, all other Hadean units behaved exactly how they are supposed to, hence why I didn't include a separate unit breakdown like the Cerberi doc.

**Comment by GraveyBoi** [comment]
Xerrakis Drone does NOT detonate against enemy units/buildings (tried both). Can hear detonation firing sound, and the "mine" does damage against building but the unit itself does not detonate, thus defeating the purpose.   --- interesting, the Xeraxis in VSR works properly.

**Comment by Malifious** [comment]
> 
> 
> --- interesting, the Xeraxis in VSR works properly.

Yeah I thought that was strange because I didn't have a problem with the Hadeans in VSR either. Tried it on both units and various buildings.

**Comment by jack775544** [comment]
Xenomator not having a console is on purpose. The empty slot for one is built into the model if some other mod wants one in the future.

**Comment by BlackDragonN001** [comment]
Defender ROF is double because of ShotAlternate, so it still fires same amount of shots, just alternates now.

Believers Vat: Probably needs hp_vehicle. Will notify Jayden.

Fixed slag mortar, and flame mine, and xarrakis.

Original FE didn't have any special icon for Xyr Guard SR, but i can get Jayden to make one.

**Comment by BlackDragonN001** [comment]
Hadean side of this should be fixed. How much of these are still valid? Cerberi remodel is still pending, so model/collision issues are TBA.
